blog.html

```diff
@@ -22,6 +22,286 @@
+                        duration based offset reset strategy. This KIP allows applications to consume/initialise from a fixed
+                        duration when there is no initial offset in Kafka.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/XoowEg">KIP-1043: Administration of groups</a>
+                        <br>
+                        KIP-848 and KIP-932 introduce new group types, consumer and share respectively. The Admin Client
+                        API has limitations when dealing with newer group types, sometimes returning errors or incorrectly
+                        stating that groups don't exist. To address this challenge, a new command-line tool, <code>kafka-groups.sh</code>,
+                        and updates to existing command line tools, <code>kafka-consumer-groups.sh</code> and <code>kafka-share-groups.sh</code>,
+                        were made in Kafka 4.0 to enable users to view all groups in a cluster, along with their types and
+                        protocols, providing accurate information even when the Admin Client API fails.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/w4mMEw">KIP-1099: Extend kafka-consumer-groups command line tool to support new consumer group</a>
+                        <br>
+                        This KIP extends the <code>kafka-consumer-groups.sh</code> and the <code>kafka-share-groups.sh</code> command line tools to
+                        provide more information related to consumer and share groups. This helps when it comes to
+                        troubleshooting consumer and share groups, respectively introduced by KIP-848 and KIP-932.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/KYuMEw">KIP-1102: Enable clients to rebootstrap based on timeout or error code</a>
+                        <br>
+                        KIP-1102 enhances Kafka client resilience by proactively triggering metadata rebootstrap when no
+                        updates occur within a timeout period and allowing servers to explicitly signal clients to rebootstrap.
+                        This addresses previous limitations in KIP-899, where clients could become stuck with outdated
+                        metadata unless all brokers were unreachable.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/K5sODg">KIP-896: Remove old client protocol API versions in Kafka 4.0</a>
+                        <br>
+                        For the first time, old protocol API versions have been removed. Users should ensure brokers are
+                        version 2.1 or higher before upgrading Java clients (including Connect and Streams) to 4.0. Similarly,
```

Comment:
Is there something specific about clients of Java language here? Can we simply say clients?

Reply:
Ah, I believe that we have no way to confirm if other clients follow the same versioning, so we cannot say generic client version 2.1.

Perhaps, we can add something like, "Clients other than Java should use API version 2.1 or higher to connect to 4.0 brokers"? Feel free to re-phrase and not a blocking comment. Upto you to decide if we want to include this information.